test/Microsoft.AspNetCore.Authorization.Test/PolicyEvaluatorTests.cs

```diff
@@ -104,6 +105,30 @@ public class PolicyEvaluatorTests
         }
 
         [Fact]
+        public async Task Challenge()
```

Comment:
Aren't these for a different issue?

Reply:
Oops yeah was just testing stuff, didn't mean to check these in, good catch, will remove